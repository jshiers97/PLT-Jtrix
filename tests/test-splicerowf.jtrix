void main() {
	Matrix<float> a;
	float[] b;
	int i;
	int j;

	a = [[1., 2.];[3., 4.]];
	a = a.spliceRow_f(0);

	b = a.dim_f();

	for(i = 0; i < f_to_int(b[0]); i = i++) {
		for(j = 0; j < f_to_int(b[1]); j = j++) {
			printf(a[i][j]);
		}
	}

	free(b);
	free(a);
}
