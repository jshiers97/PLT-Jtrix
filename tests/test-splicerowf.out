3.000
4.000
